src/platform/web/ui/session/settings/SettingsView.js

```diff
@@ -90,6 +90,18 @@ export class SettingsView extends TemplateView {
             })
         );
 
+        settingNodes.push(
+            t.h3("Theme"),
+            ...vm.themes.map(name => {
```

Comment:
would be nicer to have this as a `<select>` rather than a set of radio buttons

Reply:
Makes sense. I changed it around to this:

![Screenshot 2021-05-18 at 20 48 05](https://user-images.githubusercontent.com/1137962/118707137-76662c00-b81a-11eb-929b-5e9a2ab29e88.png)